### PR TITLE
Write Demo help component

### PR DESCRIPTION
**Pull request by jonmattgray**
Add a help component for both 'old' and 'modern' view of the write demo.

image
image
image

### CODE REVIEW

File: src/components/demos/write/Demo.js [hunk 1]
```diff
@@ -21,6 +22,13 @@ const Wrapper = styled.div`
   ${(props) => props}
 `
 
+const helpContent = `
```
Comment (n3op2) [inline src/components/demos/write/Demo.js:25]: could this be in `fixtures/demo.js`

File: src/components/shared/Help.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  position: absolute;
+  top: 200px;
+  left: 50px;
+`
+
+const Button = styled.div`
```
Comment (n3op2) [inline src/components/shared/Help.js:10]: could this also be moved into `shared/Button.js` as maybe icon button

File: src/components/demos/write/Demo.js (hunk 1 above)
Comment (jonmattgray) [reply src/components/demos/write/Demo.js]: done

File: src/components/shared/Help.js (hunk 2 above)
Comment (jonmattgray) [reply src/components/shared/Help.js]: done

**Review by mattdean-digicatapult — APPROVED** [review]
lgtm

**Pull request merged**